sns-chops/resolution
# Try to get it to work with new dash

To test:
- Go to the main folder. Use the environment.yaml file to create a new environment:  `conda env create`
- activate the environment `conda activate resolution`
- CD into the `dashui` and run `python app.py`
- see http://localhost:8050/

The convolution is not working (plan to remove it)
